slices/debconf.yaml

```diff
@@ -0,0 +1,44 @@
+package: debconf
+
+essential:
+  - debconf_copyright
+
+slices:
+  bash-completion:
+    contents:
+      /usr/share/bash-completion/completions/debconf:
+      /usr/share/bash-completion/completions/debconf-show:
+
+  bins:
+    essential:
+      - debconf_config
+      - debconf_modules
+    contents:
+      /usr/bin/debconf:
+      /usr/bin/debconf-apt-progress:
+      /usr/bin/debconf-communicate:
+      /usr/bin/debconf-copydb:
+      /usr/bin/debconf-escape:
+      /usr/bin/debconf-set-selections:
+      /usr/bin/debconf-show:
+      /usr/sbin/dpkg-preconfigure:
+      /usr/sbin/dpkg-reconfigure:
+
+  config:
+    contents:
+      /etc/apt/apt.conf.d/70debconf:
+      /etc/debconf.conf:
+
+  modules:
```

Comment:
this slice has a mix of actual modules, with helper scripts (e.g. /usr/share/debconf/confmodule), obsolete scripts (e.g. /usr/share/debconf/confmodule.sh), configurations (e.g. /usr/share/debconf/frontend), etc.

For the sake of getting this PR in asap:
 - do we actually need `debconf` anywhere? I don't see it as a dependency for anything else at quick glance. If we don't, let's leave it out and address it in a separate PR
 - if we do, do we need these modules? all of them? (same for binaries)
 - ultimately, these modules seem to be a mixed pot of things. So can we split it further?

Finally, if this SDF really needs to go in, it also needs tests

Reply:
The debconf slice isn't needed anymore after adding the mutate script in libpam-runtime, so I'll remove it.